Packs/EWS/Integrations/SecurityAndCompliance/SecurityAndCompliance.ps1

```diff
@@ -0,0 +1,1545 @@
+. $PSScriptRoot\CommonServerPowerShell.ps1
+
+$script:INTEGRATION_NAME = "Security And Compliance"
+$script:COMMAND_PREFIX = "o365-sc"
+$script:INTEGRATION_ENTRY_CONTEX = "O365.SecurityAndCompliance.ContentSearch"
+$script:SEARCH_ENTRY_CONTEXT = "$script:INTEGRATION_ENTRY_CONTEX.Search(val.Name && val.Name == obj.Name)"
+$script:SEARCH_ACTION_ENTRY_CONTEXT = "$script:INTEGRATION_ENTRY_CONTEX.SearchAction(val.Name && val.Name == obj.Name)"
+
+#### HELPER FUNCTIONS ####
+
+function UpdateIntegrationContext([OAuth2DeviceCodeClient]$client){
+    $integration_context = @{
+        "DeviceCode" = $client.device_code
+        "DeviceCodeExpiresIn" = $client.device_code_expires_in
+        "DeviceCodeCreationTime" = $client.device_code_creation_time
+        "AccessToken" = $client.access_token
+        "RefreshToken" = $client.refresh_token
+        "AccessTokenExpiresIn" = $client.access_token_expires_in
+        "AccessTokenCreationTime" = $client.access_token_creation_time
+    }
+
+    $Demisto.setIntegrationContext($integration_context)
+    <#
```

Comment:
docstrings should be at the beginning of the function (relevant for all functions)

Reply:
As we spoke, The docs string should be in the lower part of the integration according to the [official docs](https://docs.microsoft.com/en-us/powershell/scripting/developer/help/examples-of-comment-based-help?view=powershell-7.1#example-2-comment-based-help-for-a-script).